packages/editor/src/browser/navigation/navigation-location-service.ts

```diff
@@ -164,6 +164,14 @@ export class NavigationLocationService {
         return this._lastEditLocation;
     }
 
+    /**
+     * Clears the location stack and resets the pointer.
+     */
+    clearHistory(): void {
+        this.stack = [];
+        this.pointer = -1;
```

Comment:
do we need to unset `this._lasteditorLocation` as well?

Reply:
sounds good, I reset it !